lib/cadence/client/thrift_client.rb

```diff
@@ -97,13 +97,14 @@ def start_workflow_execution(
         send_request('StartWorkflowExecution', request)
       end
 
-      def get_workflow_execution_history(domain:, workflow_id:, run_id:)
+      def get_workflow_execution_history(domain:, workflow_id:, run_id:, next_page_token: nil)
         request = CadenceThrift::GetWorkflowExecutionHistoryRequest.new(
           domain: domain,
           execution: CadenceThrift::WorkflowExecution.new(
             workflowId: workflow_id,
             runId: run_id
-          )
+          ),
+          nextPageToken: next_page_token
```

Comment:
is mixing the case intentional here?

Reply:
@cedric-cordenier yes, Cadence Thrift definitions all use camelCase :/